### PR TITLE
JBIDE-11436 JBIDE-19826 update to use new lic...
...ense terms (new address); clean up site/category.xml

### PR DESCRIPTION
**Pull request by nickboldt**
add org.jboss.tools.foundation.license.feature.source

### CODE REVIEW

**Comment by fbricon** [comment]
+1 if you remove the pointless source feature

**Comment by mickaelistria** [comment]
What about amending previous commit rather than adding a fix commit? A beauty of code review is that we don't need to keep such level of detail in the actual history.

**Comment by nickboldt** [comment]
I just wanted to show you my changes. But yes, I can squash. #thereISquashedIt

**Comment by mickaelistria** [comment]
+1 to merge it. You can also add an "Also-By: Nick Boldt <nboldt@redhat.com>" line to the commit message if you want.

File: site/category.xml [hunk 1]
```diff
@@ -1,155 +1,65 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <site>
-	<description>
-	To install these features, point Eclipse at this site.
-	</description>
+	<description>To install these features, point Eclipse at this site.</description>
 
-	<category-def label="JBoss Tools Base Nightly Build Update Site" name="AllFeatures">
```
Comment (maxandersen) [inline site/category.xml]: are these new reformatting just for show or have some meaning for this ?

Comment (nickboldt) [inline site/category.xml]: reduction of number of lines and removal of pointless metadata such as:

    url="features/org.jboss.tools.test.feature.source_0.0.0.jar" version="0.0.0"

**Comment by nickboldt** [comment]
@alexeykazakov @dgolovin @maxandersen can you merge this or explain why we shouldn't?

**Comment by mickaelistria** [comment]
+1 to merge it ASAP

**Comment by maxandersen** [comment]
pushed to master.

**Pull request merged**